site/docs/flink.md

```diff
@@ -301,7 +301,22 @@ Table create commands support the most commonly used [flink create clauses](http
 * `COMMENT 'table document'` to set a table description.
 * `WITH ('key'='value', ...)` to set [table configuration](./configuration.md) which will be stored in apache iceberg table properties.
 
-Currently, it does not support computed column, primary key and watermark definition etc.
+Currently, it does not support computed column and watermark definition etc.
+
+### `PRIMARY KEY`
```

Comment:
Could you add  description of `PRIMARY KEY`, below line 300?

Reply:
ok